Mighty-Menorahs/reactive-nutshell
# Jb style eventand authorization

# Description

Styling and A lot of other stuff! Auth is working as well thanks to Jisie and Joe! It's great!

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


- [x ] New feature (non-breaking change which adds functionality)


# Testing Instructions

1. Git Checkout Master
2. Git Fetch --all
3. Git Pull Origin master
4. Checkout my branch jbStyleEventandAuthorization
5. See if my styling is on Events and see if You can Use SignUp and if the NavBar refreshes!



# Checklist:
